package.json

```diff
@@ -1,37 +1,50 @@
 {
   "name": "@open-formulieren/formio-renderer",
-  "version": "1.0.0",
+  "version": "0.1.0",
   "description": "Alternative Form.io renderer made for Open Forms.",
-  "author": "",
-  "license": "EUPL-1.2",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/open-formulieren/formio-renderer.git"
-  },
-  "main": "dist/index.js",
-  "module": "dist/index.modern.js",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "types": "./dist/esm/index.d.ts",
   "source": "src/index.tsx",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js",
+      "default": "./dist/cjs/index.js"
+    },
+    "./esm/": "./dist/esm/"
+  },
   "scripts": {
-    "build": "microbundle-crl --no-compress --format modern,cjs",
-    "start": "microbundle-crl watch --no-compress --format modern,cjs",
-    "test": "run-s test:unit test:lint test:build",
-    "test:build": "run-s build",
-    "test:lint": "eslint .",
-    "test:unit": "cross-env CI=1 react-scripts test --env=jsdom",
-    "test:watch": "react-scripts test --env=jsdom",
-    "predeploy": "cd example && yarn install && yarn run build",
-    "deploy": "gh-pages -d example/build",
-    "storybook": "start-storybook -p 6006",
+    "build:esm": "tsc",
+    "build:cjs": "tsc --module commonjs --outDir dist/cjs",
     "build:storybook": "build-storybook --webpack-stats-json",
+    "build": "npm-run-all build:*",
+    "test": "run-s test:lint",
+    "test:lint": "eslint src/**/*.ts[x]",
+    "checkformat": "prettier --check 'src/**/*.{js,scss,ts,tsx,mdx}'",
     "format": "prettier --write 'src/**/*.{js,scss,ts,tsx,mdx}'",
-    "checkformat": "prettier --check 'src/**/*.{js,scss,ts,tsx,mdx}'"
+    "start": "start-storybook -p 6006",
+    "storybook": "start-storybook -p 6006"
+  },
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/open-formulieren/formio-renderer.git"
   },
+  "author": "Maykin Media",
+  "license": "EUPL-1.2",
+  "bugs": {
+    "url": "https://github.com/open-formulieren/formio-renderer/issues"
+  },
+  "homepage": "https://github.com/open-formulieren/formio-renderer#readme",
   "peerDependencies": {
+    "formiojs": "~4.13.0 || ~4.14.0",
```

Comment:
this should work for the formio utilities aspect without forcing the entire library as a CJS module in this library. Hopefully this gives us enough flexibililty to sort out the tree shaking.